File: Archiving-Java/app/src/main/java/com/tokbox/sample/archiving/network/GetSessionResponse.java
package com.tokbox.sample.archiving.network;

import com.squareup.moshi.Json;

public class GetSessionResponse {
    @Json(name = "apiKey") public String apiKey;
    @Json(name = "sessionId") public String sessionId;
    @Json(name = "token") public String token;
}

File: Archiving-Kotlin/app/src/main/java/com/tokbox/sample/archiving/network/GetSessionResponse.kt
package com.tokbox.sample.archiving.network

import com.squareup.moshi.Json

class GetSessionResponse {

    @Json(name = "apiKey")
    var apiKey: String = ""

    @Json(name = "sessionId")
    var sessionId: String = ""

    @Json(name = "token")
    var token: String = ""
}